roles/rke2_common/tasks/config.yml

```diff
@@ -14,45 +14,21 @@
     owner: root
     group: root
 
+- name: Set a fact
+  ansible.builtin.set_fact:
+    all_node_labels: "{{ rke2_config['node-label'] | default([]) }} + {{ extra_node_labels | default([]) }}"
+
+- name: Add node labels to rke2_config
+  ansible.utils.update_fact:
```

Comment:
Greetings,
I think that this will break the "Deployment environment must have Ansible 2.9.0+" requirement.

https://docs.ansible.com/ansible/latest/collections/ansible/utils/update_fact_module.html

This hurts those of us on RHEL. 

Thanks.